### DISCUSSION TITLE
Ayuda para unirse a teams

### DISCUSSION THREAD

**Discussion by safraZX** [Launch-X-Latam/MisionFrontEnd#531] [Team Posts]
Hola, alguien sabe como me puedo unir a back-end en teams, ya he ingresado a la comunidad de gitHub pero no se como tenerla en teams, gracias

**Comment by gersondlarosa** [comment]
yo aun no tengo el correo podrían apoyarnos

**Comment by Amhedriel** [comment]
En mi caso me enviaron un par de correos el Sábado y ayer para ingresar, revisen sus buzones que por ahi les debieron enviar los enlaces.

Si no lo encuentran pregunten a soporte.

Ahora si están en TEAMS, diríjanse al apartado de Equipos, en la parte de arriba Todos los Equipos, busquen el de BackEnd y listo, saludos.
